### PR TITLE
XS-3172: Add test cases to travisCI

### PR DESCRIPTION
**Pull request by jasonleinbach-wf**
#### Changes:

- Add v66 of the test cases and added a runner script to run them
- Add a python script to parse the output
- Add to travisCI
- Excluded DQCs 13, 14, and 15 since they take forever currently.  Follow-ups written.
- Had to exclude a few failing tests in the python script.  We should investigate these.

Note: Go commit-by-commit in review.  Way too many filings added.

Wrote two follow-ups:
https://jira.atl.workiva.net/browse/XS-3194 - to get us on the latest test cases
https://jira.atl.workiva.net/browse/XS-3195 - to include DQCs 13, 14, and 15

Please review: @hefischer  @andrewperkins-wf

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,3 +7,4 @@ install: "pip install -r requirements-dev.txt"
 script: 
   - nosetests tests.unit_tests
   - flake8 dqc_us_rules tests
+  - travis_wait 30 ./run_integration.sh
```
Comment (sagesmith-wf) [inline .travis.yml]: Git wants newline

File: untitled.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+# Run DQC Conformance Suite tests
+TESTCASESROOT=/Users/jasonleinbach/Downloads/DQC_Testcases_v66
+OUTPUTLOGFILE=/Users/jasonleinbach/DQC-v70-log.xml
+OUTPUTERRFILE=/Users/jasonleinbach/DQC-v70-err.txt
+OUTPUTCSVFILE=/Users/jasonleinbach/DQC-v70-report.csv
+TESTCASESINDEXFILE="$TESTCASESROOT/index.xml"
+rm $OUTPUTLOGFILE $OUTPUTERRFILE
+python3.4 -m arelle.CntlrCmdLine --file "$TESTCASESINDEXFILE" --validate --plugins '~/workspaces/wf/dqc_us_rules/dqc_us_rules|validate/EFM|logging/dqcParameters.py' --csvTestReport "$OUTPUTCSVFILE"  --logFile "$OUTPUTLOGFILE" --disclosureSystem efm-pragmatic-all-years --logCodeFilter '(?!EFM)' 2>  "$OUTPUTERRFILE"
```
Comment (sagesmith-wf) [inline untitled.sh]: git wants a newline

Comment (jasonleinbach-wf) [inline untitled.sh]: This file was later replaced.

File: .travis.yml (hunk 1 above)
Comment (jasonleinbach-wf) [inline .travis.yml]: addressed

File: untitled.sh (hunk 2 above)
Comment (jasonleinbach-wf) [inline untitled.sh]: addressed in replacement file.

**Comment by sagesmith-wf** [comment]
+1 Pending travis

**Comment by sagesmith-wf** [comment]
+1

**Comment by brianmyers-wf** [comment]
+1

**Comment by sagesmith-wf** [comment]
+10 Verified that the integration tests were added

**Comment by andrewperkins-wf** [comment]
Reviewed and QAed - merging.

**Pull request merged**